user/views.py

```diff
@@ -1,3 +1,5 @@
+from json import loads
```

Comment:
just use `import json`. a method `loads` is a too generic name, we always import json and call function onto the module.